src/Compilers/CSharp/Test/Syntax/Parsing/RoleParsingTests.cs

```diff
@@ -0,0 +1,1206 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests;
+
+public sealed class RoleParsingTests : ParsingTests
+{
+    private new SyntaxTree UsingTree(string text, params DiagnosticDescription[] expectedErrors)
+        => UsingTree(text, TestOptions.RegularNext, expectedErrors);
+
+    private void UsingNode(string text, CSharpParseOptions? options = null, DiagnosticDescription[]? expectedParsingDiagnostics = null, DiagnosticDescription[]? expectedBindingDiagnostics = null)
```

Comment:
>UsingNode [](http://example.com/codeflow?start=17&length=9)

Consider removing if not used.

Reply:
It was used in just one place. Switch to use UsingTree instead